src/views/CourseMap/components/CourseGraph.tsx

```diff
@@ -6,17 +6,29 @@ interface CourseGraphProps {
   courses: Course[];
   onCourseClick: (courseIds: number) => void;
   selected: number;
+  borderColor?: string;
+  size?: number;
+  arrow?: boolean;
+  arrowType?: string;
 }
 
-export default function CourseGraph({ courses, onCourseClick, selected }: CourseGraphProps) {
+export default function CourseGraph({
+  courses,
+  onCourseClick,
+  selected,
+  borderColor = '#000000',
+  size = 100,
+  arrow = true,
+  arrowType = 'arrow',
+}: CourseGraphProps) {
   const [state, setState] = useState({
     graph: {
       nodes: [
         { id: 1, label: 'CS31' },
         { id: 2, label: 'CS32' },
         { id: 3, label: 'CS33' },
         { id: 4, label: 'CS35L' },
-        { id: 5, label: 'PHYS1A' },
+        { id: 5, label: 'PHYS1longtexttesttest' },
```

Comment:
I think it's a good idea to test this. The current text wrapping is not the prettiest, but it seems reasonable. I would test with longer strings with spaces from Figma like "PREPARATION FOR THE MAJOR" or "AMERICAN HISTORY AND INSTITUTIONS". Ideally, it matches the Figma text wrapping, but honestly, it's a very minor detail.